{
    "type": "bundle",
    "id": "bundle--d7a46f8a-38a5-4977-b501-39103d563c8a",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--7ae0e3e6-91fd-4453-8fde-ea3e992ff7bc",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-05-09T00:05:20.59387Z",
            "modified": "2023-05-09T00:05:20.59387Z",
            "name": "CVE-2023-27966",
            "description": "The issue was addressed with improved checks. This issue is fixed in macOS Ventura 13.3. An app may be able to break out of its sandbox",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-27966"
                }
            ]
        }
    ]
}